src/components/BoardNewColumn/BoardNewColumn.css

```
@@ -0,0 +1,16 @@
+.board-new-column {
+  width: 100%;
+  height: 100%;
+  background-color: var(--color-cloud-gray);
+  color: var(--color-medium-grey);
```

El color no se esta aplicando al texto, sugiero agregarlo aumentando especificidad antes del hover para evitar pisarlo

```suggestion
.board-new-column.board-new-column__label {
  color: var(--color-medium-grey);
}
```